### DISCUSSION TITLE
基于v2ray V5重构？

### DISCUSSION THREAD

**Discussion by ttc0419** [XTLS/Xray-core#1673] [General]
v2ray 5.1.0以后正式配置文件稳定。先前的XTLS协议正好也要删除，请问有计划基于v2ray V5的重构么？

**Comment by FranzKafkaYu** [comment]
两家分化已经有很多不同了，即使重构也没必要依照v2ray V5版本进行重构。

**Reply by ttc0419** [reply]
主要怕以后geoip文件格式一些的东西有breaking change到时候重构更加麻烦。毕竟xray主要的核心还是新协议上，基础代码库的开发不是重点。当然可以继续用老的，只不过到时候要自己维护一套。

**Comment by mclovin-2k** [comment]
要看主要的几个贡献者愿不愿意了
